_lifeofinternatpuppet/post_27.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: page
+title: Week 27
+author: Disha-maker
+categories:
+  - team
+  - basics
+tags:
+  - week 27
+---
+
+Hi everyone,
+
+- I was back to work on Monday 15th March, and released lots of modules using the new release process learnt before going on PTO. I also learnt that JIRA tickets will no longer be raised with the new release process
+
+**Release Prep PRs**
+    [puppetlabs-accounts](https://github.com/puppetlabs/puppetlabs-accounts/pull/371) (v7.0.1)
+    [puppetlabs-apt](https://github.com/puppetlabs/puppetlabs-apt/pull/977) (v8.0.1)
+    [puppetlabs-reboot](https://github.com/puppetlabs/puppetlabs-reboot/pull/285) (v4.0.1)
+
+**Module Releases**
+    [puppetlabs-accounts](https://forge.puppet.com/puppetlabs/accounts) (v7.0.1)
+    [puppetlabs-apt](https://forge.puppet.com/puppetlabs/apt) (v8.0.1)
+    [puppetlabs-reboot](https://forge.puppet.com/puppetlabs/reboot) (v4.0.1)
+
+I really enjoyed releasing these modules :)
+
+- I was also told by Paula during the 1:1 session that start putting the content of the Google Doc - 'Accessing IAC Training Materials' on Confluence. I also learnt that all training materials are drafted and reviewed on Confluence. Thanks for pointing out Paula :)
+
+- The first draft of Accessing IAC Training Materials on Confluence was ready by the end of the week!
+
+- I also spent time catching up on my blog posts for the 'Life of an Intern at Puppet' blog page for Weeks 17 and 18. The [week 17 PR](https://github.com/puppetlabs/iac/pull/205) was merged by Paula, thank you!
+
+
```

Comment:
Large gap here I think that may be an accident.

Reply:
This is also fixed in the commit message below, check the files please.